src/amo/components/SearchForm/styles.scss

```diff
@@ -17,7 +17,7 @@
   @include padding-end(6px);
   @include padding-start(28px);
 
-  border: 0;
+  border: 1px solid $teal-50;
```

Comment:
`$teal-50` should only be on focus - but a default border will be needed too - probably transparent or the same as the current input background.

Reply:
Sorry I realise I screwed up my description above and didn't qualify that teal-50 was for the focus style only. I've edited that now.